SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

We do not have /dev at this stage of the bootstrap, including /dev/tty. For
some reason, bash has a fixation on /dev/tty, even though we are not
interactive. Removing this check entirely fixes this issue.

diff --color -ru shell.c
--- bash-2.05b/shell.c	2002-07-02 01:27:11.000000000 +1000
+++ bash-2.05b/shell.c	2021-01-16 11:23:36.407287955 +1100
@@ -342,8 +342,6 @@
 #  endif
 #endif
 
-  check_dev_tty ();
-
 #ifdef __CYGWIN__
   _cygwin32_check_tmp ();
 #endif /* __CYGWIN__ */
